openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -37,20 +32,24 @@ def process(self, instance):
             self.log
         )
 
-        preset = lib.load_capture_preset(data=capture_preset)
+        # Create temp directory for thumbnail
+        # - this is to avoid "override" of source file
+        dst_staging = tempfile.mkdtemp(prefix="pyblish_tmp_")
```

Comment:
Might be worth having a more explicit prefix `pyblish_extract_thumbnail` ? I think the `tmp` is redundant since it will be already in the `tmp` directory?